rikili/ProPlanner
# PLAN-18 Style trip calendar and connection to backend

Jira Ticket: https://pro-planner.atlassian.net/browse/PLAN-18

Features covered in this PR:
* connecting trip calendar to associated API calls in backend
* styling for calendar component
* month selector to allow easier navigation between months
* detailed list of users who selected certain days (based on user click input)
* certain failure cases for not loading data
* multipurpose loading component

To test out this PR:
Make sure you have the backend and frontend running with associated `.env` files filled with the required URLs. Create a new plan or use the code for an existing trip plan to view the calendar. Since multiple users are supported in the backend yet, there are a few existing trips with mock, multiple user data.
